doc/ko/README-ko.md

```diff
@@ -0,0 +1,147 @@
+```
+
+이 경우에 결과물은 다음과 같을 것입니다(불규칙 변동으로 인해):
+
+```
+COMPLETED
+{'00': 509, '11': 515}
+```
+여러분은 또한 QISKit을 사용하여 여러분의 코드를 [real Quantum Chip](https://github.com/QISKit/ibmqx-backend-information)에서 실행시킬 수 있습니다.
+
+ 먼저, API 토큰을 얻으세요:
+ 
+-  [IBM Q experience](https://quantumexperience.ng.bluemix.net) 계정을 생성하세요. 
+-  IBM Q experience 웹 페이지에서 API token을 얻으세요. 토큰은 “My Account” > “Personal Access Token”에 위치해 있습니다. 
+
+이 API 토큰은 여러분의 프로그램을 IBM Q experience 백엔드 환경에서 실행할 수 있도록 해줄 것입니다. [예제](doc/example_real_backend.rst).
+
+이에 대해 더 많은 정보를 알고 싶다면, 다음의 링크를 참조하세요. [our QISKit documentation](doc/qiskit.rst).
+
+
+### Next Steps
+
+이제 여러분은 다른 [튜토리얼](https://github.com/QISKit/qiskit-tutorial)들까지도 실행할 수 있는 준비와 환경 구성을 마쳤습니다. [index tutorial](https://github.com/QISKit/qiskit-tutorial/blob/master/index.ipynb)을 시작하세요. 그리고 [‘Getting Started’ 예제](https://github.com/QISKit/qiskit-tutorial/blob/002d054c72fc59fc5009bb9fa0ee393e15a69d07/1_introduction/getting_started.ipynb)로 이동하세요. 만일 여러분이 이미 [Jupyter Notebooks](https://jupyter.readthedocs.io/en/latest/install.html)을 설치했다면, 여러분은 해당 노트북을 복사하고 수정하여 여러분 만의 노트북을 만들 수 있습니다. 
+
+튜토리얼을 QISKit의 일부로서 설치하기 위해 다음의 링크를 확인하세요. [installation details](doc/install.rst#Install-Jupyter-based-tutorials). 완전한 SDK문서는 다음의 링크에서 확인할 수 있습니다. [*doc* directory](doc/qiskit.rst).
+
+## More Information
+
+다음은 QISKit을 어떻게 써야하는지에 대한 더많은 정보와 튜토리얼 예제, 그리고 몇몇의 도움이 될만한 링크들 입니다. 한 번 살펴보세요. 
+* **[User Guides](https://github.com/QISKit/ibmqx-user-guides)**,
+  양자정보와 양자컴퓨팅에 대해 배울 수 있는 좋은 시작점 
+* **[Tutorials](https://github.com/QISKit/qiskit-tutorial)**,
+  예를 들어 Jupyter 노트북의 경우 [index](https://github.com/QISKit/qiskit-tutorial/blob/master/index.ipynb)를 참고하세요. 그리고 [‘Getting Started’ Jupyter notebook](https://github.com/QISKit/qiskit-tutorial/blob/002d054c72fc59fc5009bb9fa0ee393e15a69d07/1_introduction/getting_started.ipynb)도 함께 보세요. 
+* **[OpenQASM](https://github.com/QISKit/openqasm)**,
+  QASM에 대한 추가적인 정보와 예제
+* **[IBM Quantum Experience Composer](https://quantumexperience.ng.bluemix.net/qx/editor)**,
+  실제 양자컴퓨터 및 시뮬레이션된 양자컴퓨터와의 GUI 인터페이스를 통한 인터렉션
+* **[QISkit Python API](https://github.com/QISKit/qiskit-api-py)**, 파이썬을 통해 IBM Quantum Experience를 사용할 수 있는 API 
+
+QISKit은 본래 [IBM Research](http://www.research.ibm.com/)연구팀과 [IBM-Q](http://www.research.ibm.com/ibm-q/)개발팀에 의해 양자컴퓨터에 대한 고수준(high level) 개발킷을 제공할 목적으로 개발되었습니다. 
+질의응답과 토론 그리고 양자컴퓨팅에 대해 더 넓게 살펴보기 위해 [IBM Q experience community](https://quantumexperience.ng.bluemix.net/qx/community)를 방문하세요. 만일 여러분이 QISKit에 기여하길 원한다면 다음의 가이드라인을 살펴보세요. [contribution guidelines](CONTRIBUTING.rst).
+
+## Multilanguage guide
+
+* **[Korean Translation](https://github.com/QISKit/qiskit-sdk-py/tree/master/doc/ko/README-ko.md)**, 한글 기본 가이드 라인
+
+## Authors (alphabetical)
+
+Jim Challenger, Andrew Cross, Vincent Dwyer, Mark Everitt, Ismael Faro, Jay Gambetta, Juan Gomez, Paco Martin, Antonio Mezzacapo, Jesus Perez, Russell Rundle, Todd Tilma, John Smolin, Erick Winston, Chris Wood, Yunho Maeng
```

Comment:
The authors should be sorted alphabetically of their family names

Reply:
Thanks again